src/js/InvertedIndex.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * InvertedIndex class
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.files = [];
+  }
+
+  /**
+   * Validates file content
+   * @function
+   * @param {Array} fileContent content of uploaded file
+   * @return {Array|Boolean} valid fileContent or false if invalid file
+   */
+  static validateFile(fileContent) {
+    if (!(fileContent instanceof Array) || fileContent.length < 1) {
+      return false;
+    }
+
+    let invalid = false;
+    fileContent.forEach((book) => {
+      if (!(book.title && book.text)) {
+        invalid = true;
+        return;
+      }
+      if (book.text.length < 1) invalid = true;
+    });
+
+    if (invalid) return false;
```

Comment:
How about making all your conditional expressions consistent and more readable by putting their statements in a block like you did in line 27 - 30